frontend/src/download.js

```diff
@@ -29,10 +33,19 @@ $(document).ready(function() {
       // update progress bar
       $('#dl-progress').circleProgress('value', percent);
       $('.percent-number').html(`${Math.floor(percent * 100)}`);
-      $('.progress-text').append(
-        ` (${(progress[0] / 1000000).toFixed(2)}MB of ${(progress[1] /
-          1000000).toFixed(2)}MB)`
-      );
+      if (progress[1] < 1000000) {
+        $('.progress-text').html(
+          `${filename} (${(progress[0] / 1000).toFixed(1)}KB of ${(progress[1] / 1000).toFixed(1)}KB)`
```

Comment:
I think I'm getting a bit lost with all these magic numbers floating around. Not sure if we want to create some `const`s for 1000 (KB), 1000000 (MB), and 1000000000 (GB).

Something like:

```js
const KB = 1000;
const MB = KB * 1000;
const GB = MB * 1000;

if (progress[1] < MB) {
  $('.progress-text').html(
    `${filename} (${(progress[0] / KB).toFixed(1)}KB of ${(progress[1] / KB).toFixed(1)}KB)`
  );
else if (progress[1] < GB) {
```

Not sure if there'd be a prettier way to do something like:

```js
function toKB(value) {
  return `${(value / KB).toFixed(1)}KB`;
}
function toMB(value) {
  return `${(value / MB).toFixed(1)}MB`;
}
function toGB(value) {
  return `${(value / GB).toFixed(1)}GB`;
}

$('.progress-text').html(`${filename} (${toKB(progress[0])} of ${toKB(progress[1])}`);
```